<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Main</class>
 <widget class="QMainWindow" name="Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1451</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>üé´ Sistema de Gerenciamento de Tickets</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QMainWindow {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
            stop:0 #1a1a2e, stop:1 #16213e);
        color: #ffffff;
        font-family: &quot;Segoe UI&quot;, &quot;Roboto&quot;, sans-serif;
        font-size: 14px;
    }
    
    QWidget#verticalLayoutWidget {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
            stop:0 #0f3460, stop:1 #533a71);
        border-right: 2px solid #e94560;
        border-radius: 0px;
    }
    
    QPushButton {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #e94560, stop:1 #d63031);
        color: white;
        border-radius: 12px;
        padding: 12px 20px;
        margin: 8px 12px;
        font-weight: 700;
        font-size: 14px;
        border: none;
        text-align: center;
    }
    
    QPushButton:hover {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #ff6b7a, stop:1 #e74c3c);
    }
    
    QPushButton:pressed {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #c0392b, stop:1 #a93226);
    }
    
    QPushButton#pushButton_3 {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #e74c3c, stop:1 #c0392b);
        color: white;
    }
    
    QPushButton#pushButton_3:hover {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #ff4757, stop:1 #e74c3c);
    }
    
    QLabel {
        color: #ffffff;
        font-size: 16px;
        font-weight: 500;
    }
    
    QLabel#label {
        margin: 20px auto;
        padding: 10px;
        qproperty-alignment: 'AlignHCenter | AlignVCenter';
        border-radius: 20px;
        border: 3px solid #e94560;
        background-color: rgba(233, 69, 96, 0.1);
    }
    
    QLineEdit, QTextEdit, QComboBox {
        background-color: rgba(255, 255, 255, 0.1);
        border: 2px solid #e94560;
        border-radius: 8px;
        padding: 10px 15px;
        color: #ffffff;
        font-size: 14px;
        font-weight: 500;
    }
    
    QLineEdit:focus, QTextEdit:focus, QComboBox:focus {
        border: 2px solid #00d2d3;
        background-color: rgba(0, 210, 211, 0.1);
    }
    
    QLineEdit::placeholder, QTextEdit::placeholder {
        color: rgba(255, 255, 255, 0.6);
        font-style: italic;
    }
    
    QTableWidget {
        background-color: rgba(255, 255, 255, 0.05);
        color: #ffffff;
        border: 1px solid #e94560;
        border-radius: 10px;
        gridline-color: rgba(233, 69, 96, 0.3);
        font-size: 13px;
        selection-background-color: #e94560;
        selection-color: #ffffff;
        alternate-background-color: rgba(233, 69, 96, 0.1);
    }
    
    QHeaderView::section {
        background: qlineargradient(x1:0, y1:0, x2:0, y2:1, 
            stop:0 #e94560, stop:1 #d63031);
        color: #ffffff;
        padding: 12px;
        border: none;
        font-weight: bold;
        font-size: 14px;
    }
    
    QTableWidget::item {
        padding: 8px;
        border-bottom: 1px solid rgba(233, 69, 96, 0.2);
    }
    
    QTableWidget::item:hover {
        background-color: rgba(233, 69, 96, 0.2);
    }
    
    QTableWidget::item:selected {
        background-color: #e94560;
    }
    
    QStatusBar {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
            stop:0 #0f3460, stop:1 #533a71);
        color: #ffffff;
        border-top: 2px solid #e94560;
        font-size: 12px;
        font-weight: 500;
    }
    
    QToolBar {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
            stop:0 #1a1a2e, stop:1 #16213e);
        border-bottom: 2px solid #e94560;
        spacing: 8px;
    }
    
    QScrollBar:vertical {
        background: rgba(255, 255, 255, 0.1);
        width: 14px;
        border-radius: 7px;
        margin: 2px;
    }
    
    QScrollBar::handle:vertical {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
            stop:0 #e94560, stop:1 #d63031);
        min-height: 20px;
        border-radius: 7px;
        margin: 2px;
    }
    
    QScrollBar::handle:vertical:hover {
        background: qlineargradient(x1:0, y1:0, x2:1, y2:0, 
            stop:0 #ff6b7a, stop:1 #e74c3c);
    }
    
    QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        height: 0px;
    }
    
    QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical {
        background: none;
    }
   </string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">
    /* Estilos consistentes com o design principal */
    QLabel#label_4 {
        font-size: 24px;
        font-weight: bold;
        color: #00d2d3;
        text-align: center;
        padding: 10px;
        background: rgba(0, 210, 211, 0.1);
        border-radius: 10px;
        border: 2px solid #00d2d3;
    }
    </string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>60</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">
         margin: 20px auto;
         max-width: 80px;
         max-height: 80px;
         border-radius: 20px;
         border: 3px solid #e94560;
         background: qlineargradient(x1:0, y1:0, x2:1, y2:1, 
             stop:0 #e94560, stop:1 #d63031);
         padding: 10px;
        </string>
        </property>
        <property name="pixmap">
         <pixmap>ticket.png</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tel_tickets">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>üé´ Tickets</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_tela_usurios">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string>üë• Usu√°rios</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>158</width>
          <height>358</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: red</string>
        </property>
        <property name="text">
         <string>üö™ Sair</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="QTableWidget" name="tb_tickets">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>150</y>
          <width>911</width>
          <height>431</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>N¬∞</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>T√≠tulo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Cliente</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Prioridade</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Status</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Data de Cria√ß√£o</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>A√ß√µes</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="btn_novo_ticket">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>60</y>
          <width>181</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>‚ú® Novo Ticket</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="search_tickets">
        <property name="geometry">
         <rect>
          <x>620</x>
          <y>70</y>
          <width>200</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="placeholderText">
         <string>üîç Buscar tickets...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_buscar_tickets">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>60</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>üîé Buscar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QTableWidget" name="tb_users">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>150</y>
          <width>871</width>
          <height>431</height>
         </rect>
        </property>
        <column>
         <property name="text">
          <string>Nome</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>E-mail</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Tipo</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Quant. tickets</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Criado</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>A√ß√µes</string>
         </property>
        </column>
       </widget>
       <widget class="QPushButton" name="btn_novo_usuario">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>60</y>
          <width>211</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>‚ûï Adicionar Usu√°rio</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Search_user">
        <property name="geometry">
         <rect>
          <x>630</x>
          <y>70</y>
          <width>191</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
        <property name="placeholderText">
         <string>üîç Buscar usu√°rios...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_buscar_users">
        <property name="geometry">
         <rect>
          <x>830</x>
          <y>60</y>
          <width>171</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>üîé Buscar</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QPushButton" name="btn_cadastrar_usuario">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>420</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>‚úÖ CADASTRAR</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="name_user">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>210</y>
          <width>651</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>180</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>Nome</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>290</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>E-mail</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="email_user">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>320</y>
          <width>381</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>290</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>Senha</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Password_user">
        <property name="geometry">
         <rect>
          <x>510</x>
          <y>320</y>
          <width>251</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBox">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>430</y>
          <width>181</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:orange;</string>
        </property>
        <item>
         <property name="text">
          <string>Admin</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cliente</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>390</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>Tipo</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLineEdit" name="titulo_ticket">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>90</y>
          <width>651</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
       </widget>
       <widget class="QTextEdit" name="descricao_ticket">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>200</y>
          <width>651</width>
          <height>331</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>60</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>T√≠tulo</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>150</y>
          <width>131</width>
          <height>31</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>Descri√ß√£o</string>
        </property>
       </widget>
       <widget class="QPushButton" name="btn_cadastrar_ticket">
        <property name="geometry">
         <rect>
          <x>580</x>
          <y>550</y>
          <width>191</width>
          <height>61</height>
         </rect>
        </property>
        <property name="text">
         <string>üíæ Salvar Ticket</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_prioridade">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>540</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
</string>
        </property>
        <property name="text">
         <string>Prioridade</string>
        </property>
       </widget>
       <widget class="QComboBox" name="combo_prioridade">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>570</y>
          <width>171</width>
          <height>41</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color:orange;
</string>
        </property>
        <item>
         <property name="text">
          <string>Baixa</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>M√©dia</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Alta</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Urgente</string>
         </property>
        </item>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>350</x>
          <y>20</y>
          <width>281</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>-1</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:white;
background-color:transparent;
border: 2px solid red; padding: 10px;
  

</string>
        </property>
        <property name="text">
         <string>CRIAR NOVO TICKET</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
